server/api/actions/subscriptions.js

```diff
@@ -0,0 +1,58 @@
+const db = require('../../db/db');
+
+const getSubscriptionsByUserId = async (userId) => {
+  const { rows: data } = await db.execute('server/sql/subscriptions/getSubscriptionsByUserId.sql', {userId});
+  return data;
+};
+
+/*
+const getSubscriptionBySubscriptionId = async (subId) => {
+  const { rows: data } = await db.execute('../../sql/subscriptions/getSubscriptionBySubscriptionId.sql', {subId});
+  return data;
+};
+*/
+
+const createSubscription = async ({ name, nickname, due_date, reminder_days, amount, user_id }) => {
```

Comment:
other than these destructured values, are there other values in the req.body that are not being used? if not, we can use the spread operator below. feel free to msg me about that for further explanation